test/reducer.passage.test.js

```diff
@@ -68,6 +68,26 @@ describe('passageReducer', () => {
 
         expect(secondReducer.active[VERSE_MODE]).toEqual(3);
       });
+
+      it('should navigate to index', () => {
+        let initialReducer = passageReducer(initialState, {
```

Comment:
It's probably not a big deal but `initialReducer` is a misleading variable name since it refers not to a reducer but to a state. It may or may not be worth the effort to change.